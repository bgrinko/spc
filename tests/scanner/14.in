/* 13213
234324
345345 */