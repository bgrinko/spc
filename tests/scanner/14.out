Error! Line: 1 Pos: 2 Msg: Expression expected
